src/cljs/main/broadfcui/page/workspace/analysis/igv.cljs

```diff
@@ -26,7 +28,7 @@
                              {:name (str "Track " (inc index))
                               :url (common/gcs-uri->google-url track-url)
                               :indexURL (when (string? @index-url) (common/gcs-uri->google-url @index-url))
-                              :headers (user/get-bearer-token-header)
+                              :headers {"Authorization" (str "Bearer " token)}
```

Comment:
Because the default token now doesn't include storage scopes?

Reply:
correct. The default token returned by `(user/get-bearer-token-header)` is the end user's token; we need to use the pet's token that we got earlier.